examples/reference/pandas/scatter.ipynb

```diff
@@ -36,6 +36,24 @@
     "df.hvplot.scatter(x='sepal_length', y='sepal_width', by='species', \n",
     "                  legend='top', height=400, width=400)"
    ]
+  },
+  {
+   "source": [
+    "You can add fields to the hover display using the `hover_cols` argument. It can also take \"all\" as input to show all fields. "
```

Comment:
```suggestion
    "As for most other types of hvPlot plots, you can add fields to the hover display using the `hover_cols` argument. It can also take \"all\" as input to show all fields. "
```